stdlib/public/core/Collection.swift

```diff
@@ -818,6 +818,10 @@ public protocol Collection: Sequence where SubSequence: Collection {
   ///   a random element.
   /// - Returns: A random element from the collection. If the collection is
   ///   empty, the method returns `nil`.
+  ///
+  /// - Complexity: O(1) if the collection conforms to
+  ///   `RandomAccessCollection`; otherwise, O(*n*), where *n* is the absolute
+  ///   value of `n`.
```

Comment:
Shouldn't this be "where *n* is the length of the collection"? (same for the default implementations of `randomElement(using:)` and `randomElement()`) I'm not sure it's clear what `n` refers to otherwise.

Reply:
Wow, thanks — that's definitely the wrong boilerplate.